client/src/contexts/SocketContext.tsx

```diff
@@ -17,7 +19,7 @@ interface SocketContextProps {
   boardId: string | undefined;
   boardName: string | undefined;
   boardOwner: string | undefined;
-  currentPermission: string | undefined;
+  currentPermission: UserPermissions;
```

Comment:
I would change the name of this property to `currentPermissions`, because it hold a lot of different permissions now, not just one

Reply:
Well, as I mentioned in another comment 😅😅, maybe we need to distinguish actions from permissions. Otherwise, we'll just end up more confused in the future

Reply:
I commited this already, only because one convention needs to be used. But if you want to discuss this i am open to changes.